src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/entity/ApprovedDates.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotNull
+import java.time.LocalDate
+
+@Entity
+@Table
+data class ApprovedDates(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long = -1,
+  @NotNull
```

Comment:
Question about the @NotNull annotation - does it provide anything additional to the Kotlin default behaviour; i.e. the absence of a ? implies not null

Reply:
I think this needs more investigation!